### PR TITLE
Feature/custom armour cappy

### PR DESCRIPTION
**Pull request by OverYandere**
Branch includes Cappy's new requested vest. I am planning on implementing the helmet as part of separate branches.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .vscode
 build-config.json
 build/
+build.ps1
```
Comment (Lupus590) [inline .gitignore:4]: Please revert this.

**Review by Lupus590 — CHANGES_REQUESTED** [review]
Can't check everything on my phone but this is what I spotted.

We need to fix this whitespace issue, it is making PRs a pain to review.

File: .gitignore (hunk 1 above)
Comment (OverYandere) [reply .gitignore:4]: removed

**Review by Lupus590 — COMMENTED** [review]
No medic versions?

**Review by Lupus590 — APPROVED** [review]

**Pull request merged**